src/common/helpers.ts

```diff
@@ -47,6 +47,7 @@ export interface PollerInfo {
   operationName: string;
```

Comment:
With the addition of `op` I think you can remove `operationName` and `schema`.

Reply:
Done